### ISSUE TITLE
Conduct accessibility audit for Platform website

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#51108]
## User Story
_"As a VFS team member with a disability, I want to be able to access information from Platform website, so that I can build Veteran-facing products on VA.gov."_

_"As an accessibility specialist on the Platform Governance team, I want Platform's own resources to meet the accessibility standards we require of VFS teams, so that I have credibility when providing guidance to VFS teams."_

Assignee:
Peer Reviewer:

## Description
Just from a cursory review of VA.gov, there are several accessibility defects immediately apparent -- and likely many more that would be discovered from a more comprehensive audit. Although the Platform Content team has been doing a great job ensuring accessibility of content created in Confluence and published to the Platform website, other structural defects remain.

Accessibility barriers on Platform website pose two major concerns:
1. People with disabilities on VFS teams or on Platform may find it difficult to access information required to complete their work.
2. Our credibility in requiring WCAG 2.1 conformance and guidance on best practices could be called into question when the resources we create for VFS teams fall short of those standards.

The goal of this issue would be to conduct an accessibility audit with a focus on the Scroll Viewport templates rather than the page-by-page content generated in Confluence. Issues would be reported consistent with the [defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric).

Following the completion of the audit, next-steps in collaboration with the Platform Content team would focus on determining which defects can be resolved through configuration changes for Scroll Viewport, and which defects should be reported back as bugs to the Scroll Viewport support team.

## Tasks
- [ ] _Placeholder task_

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] _Placeholder AC_

### Refinement
- [ ] Ticket has user story, description, tasks, and acceptance criteria
- [ ] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [ ] Ticket has been sized 
- [ ] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [ ] Ticket has been assigned to appropriate initiative or quarterly epic
- [ ] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@vbellissimo FYI - thought this may be something your team should be made aware of and/or involved in. We have no plans at this time to complete an audit. Please let me know if you'd like to discuss further.

**Comment by shiragoodman** [comment]
@jknipes @EWashb this is another ticket that came up at Governance team refinement in January 2024. We believe it is better suited for Platform Content team, so I have removed our label and added yours.. If you disagree, or if you'd like to discuss this further and leverage some of @briandeconinck 's a11y subject matter expertise, please let us know. Thank you!

**Comment by jknipes** [comment]
Where ADE docs currently live:

![Image](https://github.com/user-attachments/assets/43d477b0-60d7-4b7c-80b2-53c3f8b9f434)
